### DISCUSSION TITLE
TD3 baseline

### DISCUSSION THREAD

**Discussion by anasbekheit** [google/brax#495] [Ideas]
Hello, firstly I want to thank the brax team for this amazing engine.
I have implemented a TD3 baseline that I used to work with and have tested it's results. it's modeled very closely to the SAC implementation provided, should I do a pull request adding the baseline?

**Comment by erikfrey** [comment]
We'd be happy to review a TD3 baseline - please also link to a colab in the PR that shows training curves so we can review the results.  Thanks!
